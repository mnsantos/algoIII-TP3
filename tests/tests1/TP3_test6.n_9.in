9 21
1 9
5 6
2 9
1 6
4 7
1 7
2 6
4 9
2 7
6 7
1 4
5 9
1 2
1 8
6 8
1 3
5 8
1 5
7 9
4 5
6 9
0
