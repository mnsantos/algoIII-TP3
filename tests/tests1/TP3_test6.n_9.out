14 3 1 6 9
